.mdformat.toml

```diff
@@ -1,3 +1,3 @@
-wrap = "keep"       # possible values: {"keep", "no", INTEGER}
+wrap = 80           # wrap all .md files to 80 characters - possible values: {"keep", "no", INTEGER} 
```

Comment:
checkers did not like the extra space at the end of this line.

Reply:
Fixed

Reply:
My branch has the commit (https://github.com/googleapis/google-cloud-cpp/compare/main...alevenberg:google-cloud-cpp:md), but it's not in the PR....

Reply:
Never mind, just a delay